docroot/sites/signage.sites.uiowa.edu/modules/sitessignage_core/sitessignage_core.module

```diff
@@ -13,21 +13,6 @@ use Drupal\user\Entity\User;
  * Implements hook_form_alter().
  */
 function sitessignage_core_form_alter(&$form, FormStateInterface $form_state, $form_id): void {
-
-  // Define the forms to remove the "Other groups" field from.
-  $og_audience_other_groups_forms = ['node_sign_form', 'node_sign_edit_form', 'node_slide_form', 'node_slide_edit_form'];
-
-  // If our form ID is in the relevant list of form IDs...
-  if (in_array($form_id, $og_audience_other_groups_forms, TRUE)) {
-
-    // If our form has other_groups defined...
-    if (isset($form['og_audience']['other_groups'])) {
-
-      // Remove access from the other groups field.
-      $form['og_audience']['other_groups']['#access'] = FALSE;
```

Comment:
It doesn't seem like this should have been removed. I'm now seeing the "Other groups" part of the OG group field widget again.

Reply:
Testing this now, I'm correctly not seeing the "other groups" section as a non-admin.

I did have some issues when using existing accounts (Marie and Brian) and trying to set them up as non-admins, either caching or lingering permissions/roles somewhere, but consistently did not see it when creating a new user with the appropriate roles.